Cargo.toml

```diff
@@ -13,3 +13,10 @@ snafu = "0.7"
 structopt = "0.3"
 tokio = { version = "1.4", features = ["full"] }
 tower = {version = "0.4"}
+directories = "4.0.1"
+reqwest = {version = "0.11", default-features = false, features = ["json", "rustls-tls", "stream"]}
+serde = "1.0.144"
+indicatif = "0.17.0"
+futures-util = "0.3.24"
+flate2 = "1.0.24"
+tar = "0.4.38"
```

Comment:
Can we re-add a newline to the end of this file please? Most editors have a setting one can set that ensures on save files end with a newline, may be worth investigating that.